pulp_smash/pulp3/utils.py

```diff
@@ -76,9 +76,9 @@ def sync(cfg, remote, repo):
         a call report.
     """
     client = api.Client(cfg, api.json_handler)
-    return client.post(
-        urljoin(remote['_href'], 'sync/'), {'repository': repo['_href']}
-    )
+    data = {'repository': repo['_href']}
+    data.update(kwargs)
+    return client.post(urljoin(remote['_href'], 'sync/'), data)
```

Comment:
pulp-smash coverage is decreasing so it would be nice if we can have a simple test case for this function added to https://github.com/PulpQE/pulp-smash/blob/master/tests/test_pulp3_utils.py

No need to perform the post (so we can use one of the mock examples) and assert that the right data is being posted.